AdvancedDLSupport/NativeLibraryBuilder.cs

```diff
@@ -117,6 +120,28 @@ public NativeLibraryBuilder
             PathResolver = pathResolver ?? new DynamicLinkLibraryPathResolver();
         }
 
+        /// <summary>
+        /// Overrides the default symbol loader for this instance of <see cref="NativeLibraryBuilder"/>.
+        /// </summary>
+        /// <param name="factory">Factory to create the overriding symbol loader.</param>
+        /// <returns>This instance of <see cref="NativeLibraryBuilder"/>.</returns>
+        public NativeLibraryBuilder WithSymbolLoader(Func<Loaders.ISymbolLoader, Loaders.ISymbolLoader> factory)
+        {
+            _customSymLoader = factory(Loaders.PlatformLoaderBase.PlatformLoader);
+            return this;
+        }
+
+        /// <summary>
+        /// Overrides the default library loader for this instance of <see cref="NativeLibraryBuilder"/>.
+        /// </summary>
+        /// <param name="factory">Factory to create the overriding library loader.</param>
+        /// <returns>This instance of <see cref="NativeLibraryBuilder"/>.</returns>
+        public NativeLibraryBuilder WithLibraryLoader(Func<Loaders.ILibraryLoader, Loaders.ILibraryLoader> factory)

```

Comment:
Write "This instance, with the library loader overridden." instead. We know it's a NativeLibraryBuilder.